tests/integration.rs

```diff
@@ -5304,6 +5304,117 @@ mod tests {
 
             handle_child_output(r, &output);
         }
+
+        #[test]
+        #[cfg(target_arch = "x86_64")]
```

Comment:
Does it need to be x86 specific?

Reply:
I tried enabling it for arm64 but the first reboot (nothing to do with the watchdog) fails.

Reply:
I will open a bug after this is merged to try and investigate this issue.